modules/crr/pages/set-up-crr.adoc

```diff
@@ -32,17 +32,65 @@ Make sure TigerGraph is *not* installed with a local loopback IP such as 127.0.0
 The following setup is needed in order to enable Cross Region Replication.
 
 
+=== Back up primary data
 
-=== Backup primary data
-
-Use GBAR to xref:backup-and-restore:index.adoc[create a backup] of the primary cluster. See xref:backup-and-restore:index.adoc[Backup and Restore] on how to create a backup.
+Use GBAR to xref:backup-and-restore:index.adoc[create a backup] of the primary cluster.
+See xref:backup-and-restore:index.adoc[Backup and Restore] for how to create a backup.
 
 If you are setting up both the primary cluster and the DR cluster from scratch, you can skip Steps 1, 2, and 4 and only perform Step 3.
 
 === Restore on the DR cluster
 
 Copy the backup files from every node to every node on the new cluster.  xref:backup-and-restore:index.adoc#_restore_from_a_backup_archive[Restore the backup] of the primary cluster on the DR cluster. See xref:backup-and-restore:index.adoc[Backup and Restore] on how to restore a backup.
 
+=== Delete metadata records
```

Comment:
Arun suggested to add a dialogue box to provide subversion details on which subversions needs to do this `Delete metadata records`. 
Currently, there is a fix that will be applied to 3.5.2+, 3.6.3+, 3.7.1+, 3.9.0+, 4.0+.
For specific subversions earlier than them, actions in this section are needed. Otherwise, user should not execute this section.

Reply:
Can you confirm that "actions in this section are needed" refers to the entire 2.3 "Delete metadata records" section?

Reply:
@alexthomasTG  Yes.